### PR TITLE
Mat 2538

### PR DESCRIPTION
**Pull request by adongare**
As per QDM 5.5 standard it appears that OID for datatype `Physical Exam Recommended` was incorrectly typed. This change is to correct it.
   
image (6)

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: MAT-2538 PhysicalExamRecommended OID is Incorrect
- [x] Internal ticket links to this PR: https://jira.cms.gov/browse/MAT-2538
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated
- [ ] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter

**Bonnie Reviewer:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Cypress Reviewer:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: dist/browser.js [hunk 1]
```diff
@@ -3413,8 +3413,8 @@ function RecursiveCast(any) {
   }
 
   if (any && any.numerator && any.denominator) {
-    numerator = new cql.Quantity(any.numerator.value, any.numerator.unit);
-    denominator = new cql.Quantity(any.denominator.value, any.denominator.unit);
+    const numerator = new cql.Quantity(any.numerator.value, any.numerator.unit);
+    const denominator = new cql.Quantity(any.denominator.value, any.denominator.unit);
```
Comment (adongare) [inline dist/browser.js:3417]: lint fixes

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cqm-models",
-  "version": "3.0.5",
+  "version": "3.0.6",
```
Comment (adongare) [inline package.json:3]: Changed version so that I don't have to create another PR to release this

File: cqm-models.gemspec [hunk 3]
```diff
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = 'cqm-models'
-  spec.version       = '3.0.5'
+  spec.version       = '3.0.6'
```
Comment (adongare) [inline cqm-models.gemspec:7]: updated the version so that I don't have to create another PR to release this

**Review by jkotanchik-SB — APPROVED** [review]

**Review by serhii-ilin — APPROVED** [review]

**Review by AndrewBird81 — APPROVED** [review]
